app/scripts/controllers/transactions/index.js

```diff
@@ -2654,4 +2654,16 @@ export default class TransactionController extends EventEmitter {
   _getApprovalId(txMeta) {
     return String(txMeta.id);
   }
+
+  _addUnapprovedTxsApprovals() {
+    const unapprovedTxs = this.txStateManager.getUnapprovedTxList();
+    Object.values(unapprovedTxs).forEach((txMeta) => {
+      this._requestApproval(txMeta, false);
+    });
+  }
+
+  addInitialTransactionApprovals(callback) {
```

Comment:
As mentioned previously, if we're not awaiting anything, then the `callback` shouldn't be needed.